sdk/src/main/java/co/omise/android/ui/PaymentChooserFragment.kt

```diff
@@ -52,7 +52,9 @@ internal class PaymentChooserFragment : OmiseListFragment<PaymentMethodResource>
             PaymentMethodResource.MaybankQR,
             PaymentMethodResource.GrabPay,
             PaymentMethodResource.PayPay,
-            PaymentMethodResource.PointsCiti -> item.sourceType?.let(::sendRequest)
+            PaymentMethodResource.PointsCiti,
+            PaymentMethodResource.GrabPay_RMS,
+            PaymentMethodResource.TouchNGo_Alipay -> item.sourceType?.let(::sendRequest)
```

Comment:
@nuxzero just to confirm, `-> item.sourceType?.let(::sendRequest)` should follow the last item which used to be `PaymentMethodResource.PointsCiti` but now is `PaymentMethodResource.TouchNGo_Alipay`, correct?

In that case, should we either define this as an `else` clause or somehow group these payment methods together?